fee-market/tests/fee_market_blackbox_test.rs

```diff
@@ -40,25 +55,638 @@ fn test_set_fee_wrong_params() {
         .common_setup
         .deploy_fee_market(Some(fee), ESDT_SAFE_ADDRESS);
 
-    state.set_fee(WRONG_TOKEN_ID, "Fixed", Some(INVALID_TOKEN_ID));
+    state.set_fee_during_setup_phase(WRONG_TOKEN_ID, WantedFeeType::Fixed, Some(INVALID_TOKEN_ID));
+
+    state.set_fee_during_setup_phase(
+        FIRST_TEST_TOKEN,
+        WantedFeeType::None,
+        Some(INVALID_FEE_TYPE),
+    );
+
+    state.set_fee_during_setup_phase(SECOND_TEST_TOKEN, WantedFeeType::Fixed, Some(INVALID_FEE));
+
+    state.set_fee_during_setup_phase(
+        FIRST_TEST_TOKEN,
+        WantedFeeType::AnyTokenWrong,
+        Some(INVALID_TOKEN_ID),
+    );
+}
+
+/// ### TEST
+/// F-MARKET_SET_FEE_FAIL
+///
+/// ### ACTION
+/// Call `set_fee()` when setup phase is not completed
+///
+/// ### EXPECTED
+/// Error CALLER_NOT_OWNER
+#[test]
+fn test_set_fee_setup_not_completed() {
+    let mut state = FeeMarketTestState::new();
+
+    state
+        .common_setup
+        .deploy_fee_market(None, ESDT_SAFE_ADDRESS);
 
-    state.set_fee(FIRST_TEST_TOKEN, "None", Some(INVALID_FEE_TYPE));
+    state
+        .common_setup
+        .deploy_header_verifier(vec![ScArray::FeeMarket, ScArray::ChainConfig]);
 
-    state.set_fee(SECOND_TEST_TOKEN, "Fixed", Some(INVALID_FEE));
+    let fee = FeeStruct {
+        base_token: FIRST_TEST_TOKEN.to_token_identifier(),
+        fee_type: FeeType::None,
+    };
 
-    state.set_fee(FIRST_TEST_TOKEN, "AnyTokenWrong", Some(INVALID_TOKEN_ID));
+    state.set_fee(&ManagedBuffer::new(), &fee, Some(CALLER_NOT_OWNER), None);
```

Comment:
This should be `SETUP_PHASE_NOT_COMPLETED` (after you remove only_owner for the function)

Reply:
Applicable for other tests below